packages/flutter/lib/src/material/material_localizations.dart

```diff
@@ -319,7 +317,7 @@ abstract class MaterialLocalizations {
   ///
   /// See also:
   ///   * [formatCompactDate], which will convert a [DateTime] into a string in the compact format.
-  DateTime parseCompactDate(String inputString);
+  DateTime? parseCompactDate(String? inputString);
```

Comment:
Why is `inputString` allowed to be null here?

Reply:
The call chain comes from `typedef FormFieldValidator<T> = String? Function(T? value);` where value is nullable.